### ISSUE TITLE
Extension Error: Failed to construct 'URL': Invalid URL

### ISSUE THREAD

**Issue by nuc** [GoogleChrome/lighthouse#1263]
**Lighthouse Version**: 1.3.1
**Chrome Version**: 57.0.2959.0
**Error Message**: Failed to construct 'URL': Invalid URL
**Stack Trace**:
 ```TypeError: Failed to construct 'URL': Invalid URL
    at artifacts.DateNowUse.usage.filter.err (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:1047:39)
    at Array.filter (native)
    at Function.audit (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:1046:48)
    at Promise.resolve.then._ (chrome-extension://blipmdconlkpinefehnmjammfjpmpbjk/scripts/lighthouse-background.js:11388:20)```

**Comment by ebidel** [comment]
Can you provide the page you were testing against and any chrome extensions that may be interfering with Lighthouse?

**Comment by brendankenny** [comment]
should be fixed with #1277 and 1.3.2
